my_list.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'my_enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
+list.each { |e| puts e }
+puts(list.all? { |e| e < 5 })
+puts(list.all? { |e| e > 5 })
+puts(list.any? { |e| e == 2 })
+puts(list.any? { |e| e == 5 })
+puts list.filter { |e| e.even? }.inspect # rubocop:disable Style/SymbolProc
```

Comment:
- [ ]  [OPTIONAL] You should not disable rubocop rules, instead look for a way to fix any linter issue related

Reply:
We will fit it, thank you.